crucible/src/Lang/Crucible/Analysis/Postdom.hs

```diff
@@ -89,7 +89,9 @@ postdomMap m breakpointIds = r
           [ (pd_id, mapMaybe f l)
           | (pd,_:l)  <- G.dom g 0
           , pd > 0
-          , let Just pd_id = Map.lookup pd idMap
```

Comment:
I think this is better replaced by a guard `Just pd_id <- Map.lookup pd idMap`.  I think those can appear in comprehensions like this (maybe with an extension enabled).

Reply:
Good catch.  I think I'd been thinking of `List.lookup`, but I agree that a linear scan of map elements is the wrong direction here.